views/admin-login.ejs

```diff
@@ -60,4 +54,10 @@
     
     </body>
 
+<script src="/js/admin-login.js"></script>
+<script src="/js/confirm-login.js"></script>
+<script src="/js/config.js"></script>
+<script src="https://www.gstatic.com/firebasejs/5.8.0/firebase.js"></script>
```

Comment:
can we safely delete the Firebase script?

Reply:
admin-login still needs firebase until Jason's work comes in